src/phpunit-command.js

```diff
@@ -20,8 +24,8 @@ module.exports = class PhpUnitCommand {
         suiteSuffix = suiteSuffix ? ' '.concat(suiteSuffix) : '';
 
         this.lastOutput = this.runFullSuite
-            ? `${this.binary}${suiteSuffix}${this.suffix}`
-            : `${this.binary} ${this.file}${this.filter}${this.configuration}${this.suffix}`;
+            ? `${this.binary}${suiteSuffix}${this.suffix}${this.coverage}`
```

Comment:
thinking "this.suffix" should probably go after "this.coverage"?

Reply:
Changed at my last commit